From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Mon, 5 Sep 2016 16:32:42 +0100
Subject: spelling

---
 swisswatch.man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/swisswatch.man b/swisswatch.man
index c6ae985..ba971df 100644
--- a/swisswatch.man
+++ b/swisswatch.man
@@ -42,7 +42,7 @@ fit the outline of the clock.
 .TP 8
 .B \-ffs
 causes the second and minute hands to behave differently at the turn
-of the minute.  This mode tries to mimick the Look and Feel of the
+of the minute.  This mode tries to mimic the Look and Feel of the
 swiss federal railway clocks.
 .SH RESOURCES
 .TP 8
